chain-cli/src/commands/deploy/index.ts

```diff
@@ -52,7 +52,7 @@ export default class Deploy extends BaseCommand<typeof Deploy> {
   async run(): Promise<void> {
     const { args, flags } = await this.parse(Deploy);
 
-    const environment = flags.testnet ? "testnet" : "mainnet";
+    const environment = flags.testnet ? "testnet" : "sandbox";
```

Comment:
double check if it does not break anything

Reply:
all good, this name is not used in the code, it's just logged to show the user which environment he wants to deploy
the deploy environment is chosen by the boolean `flags.testnet`